source/plugin/commands/childcommands.rst

```diff
@@ -0,0 +1,67 @@
+==============
+Child Commands
+==============
+
+.. javadoc-import::
+    org.spongepowered.api.command.Command.Builder
+    org.spongepowered.api.command.Command.CommandExecutor
+    org.spongepowered.api.command.exception.ArgumentParseException
+
+The :javadoc:`Command.Builder` supports hierachical command structure like this:
+
+- ``/mail`` (parent command)
+    - ``/main send`` (child command)
+    - ``/main read`` (child command)
+
+Every child command is a seperate ``Command`` and can be cheated in the same way as a regular command is.
```

Comment:
cheated -> created
(at least, I hope that's what you meant)

Reply:
Updated